lib/include/fdoprot.h

```diff
@@ -109,6 +96,7 @@
 #define FDO_TYPE_ERROR 255
 
 // Persistent
+// TODO: Needs review (Only FDO_TYPE_HMAC is used in code)
```

Comment:
should this comment be removed? What is the TODO?

Reply:
These can be removed, including the used 'FDO_TYPE_HMAC' and its usage. They don't really serve any purpose currently (unused in older Client SDK as well).

Reply:
Fixed in commit 1c24ccaba0e54c28b18fdfaa30a87b1cad086e07